README.rst

```diff
@@ -130,9 +136,35 @@ Add this to your template yaml::
                   fingerprints: 00:11:22:33:44:55:66
             - anotherteam
 
-Running this requires a github token with permissions to read the github organisation stored in an environment variable called GH_TOKEN.
-Once you set the variable just run::
+2. in the simplest version, just add the list of teams (all users' keys will be used) 
+   to the projec YAML template
+
+::
+
+
+    github_users:
+      ministryofjustice: # or any org
+        teams:
+          - webops
+          - crime-billing-online
+
+
+3. if a team doesn't exist, create it on GitHub. Example of a team:
+    https://github.com/orgs/ministryofjustice/teams/webops
+    
+
+4. obtain a GitHub token and set the GH_TOKEN variable in your environment:
+    https://help.github.com/articles/creating-an-access-token-for-command-line-use/
+
+5. run the following command:
+
+::
```

Comment:
At step 5 I would also say what the command does, ex:

This will generate `keys.sls` in pillar/env/, which you have to include in your pillar/env/top.sls for the keys to be read by salt